### DISCUSSION TITLE
使用命令行上传失败

### DISCUSSION THREAD

**Discussion by djnjbjb** [Molunerfinn/PicGo#661] [Q&A] [answered]
如果已经开启了PicGo App，使用命令行"C:\soft\PicGo\PicGo.exe" "upload" "C:\soft\PicGo\123.png"，上传就会失败。
如果没有开启PicGo App，使用命令行上传就会成功。

也许是我的用法不对？应该怎么做呢？ 

![image](https://user-images.githubusercontent.com/17374243/116021153-34faba80-a67a-11eb-8251-475246f781d6.png)

**Comment by Molunerfinn** [comment, marked as answer]
picgo相关的配置也贴出来看看，包括「上传前重命名」之类的。可以把设置页的设置截个图，回头看看复现一下。
另外使用的是什么版本？

**Reply by Molunerfinn** [reply]
可以更新一下beta.6，应该处理了windows cli上传的问题

**Comment by djnjbjb** [comment]
![image](https://user-images.githubusercontent.com/17374243/116046140-5ae98480-a6a5-11eb-9307-8b8e33150796.png)

**Reply by djnjbjb** [reply]
设置如上图，版本2.2.2

**Reply by djnjbjb** [reply]
2.3.0-beta.6 没有问题，可以上传

**Reply by djnjbjb** [reply]
再多问一个问题。
如果在PicGo的相册里吧图片删除了，在Github上不会被删除。

**Reply by Molunerfinn** [reply]
> 再多问一个问题。
> 如果在PicGo的相册里吧图片删除了，在Github上不会被删除。

不会。这个在FAQ里已经说了。如果你要同步删除，可以使用githubPlus插件

**Reply by djnjbjb** [reply]
嗯，谢啦。
抱歉没有仔细看FAQ。
